### PR TITLE
Preparing examples v3.1

### PR DESCRIPTION
**Pull request by mkarg**
Collection of useful steps towards examples v3.1

**As these are no API changes, this is a fast-track review period of just one day as per our [committer rules](https://github.com/eclipse-ee4j/jaxrs-api/wiki/Committer-Conventions#minimum-length-of-review-period-before-merge--close-of-prs-and-closing-of-issues).**

### CODE REVIEW

**Review by chkal — APPROVED** [review]

File: examples/pom.xml [hunk 1]
```diff
@@ -67,39 +61,33 @@
...
-            <name>Santiago Pericas-Geertsen</name>
-            <organization>Oracle</organization>
-            <roles>
-                <role>JAX-RS Spec Lead</role>
-            </roles>
-            <timezone>EST</timezone>
+            <id>developers</id>
+            <name>JAX-RS API Developers</name>
+            <email>jaxrs-dev@eclipse.org</email>
+            <url>https://github.com/eclipse-ee4j/jaxrs-api/graphs/contributors</url>
```
Comment (spericas) [inline examples/pom.xml:67]: Why are you doing this? This has nothing to do with "preparing examples".

**Review by spericas — CHANGES_REQUESTED** [review]

**Review by spericas — APPROVED** [review]

File: examples/pom.xml [hunk 2]
```diff
@@ -267,25 +242,25 @@
...
             <artifactId>jakarta.xml.bind-api</artifactId>
-            <version>3.0.0-RC3</version>
+            <version>3.0.1</version>
         </dependency>
 
         <dependency>
             <groupId>jakarta.enterprise</groupId>
             <artifactId>jakarta.enterprise.cdi-api</artifactId>
-            <version>3.0.0-M4</version>
+            <version>4.0.0.Alpha1</version>
```
Comment (andymc12) [inline examples/pom.xml:251]: IIUC, CDI 4.0 is part of EE 10, but Jakarta REST 3.1 should depend on EE 9/9.1 technologies.  Should this stay at CDI 3.X?

**Review by andymc12 — COMMENTED** [review]
Overall, looks good.  I think the CDI dependency should be 3.X though, and it'd be nice of the Travis CI tool ran so that we could ensure that the updated Maven tools don't break us (though I think it is unlikely to do so).  Thanks!

File: examples/pom.xml (hunk 1 above)
Comment (mkarg) [reply examples/pom.xml:67]: I have aligned it with the same text found at `jaxrs-api/pom.xml` and wonder what your actual problem is, as I do see this alignment as a step towards "preparing the examples folder". Why do you want it differently for examples than for jaxrs-api POM?

File: examples/pom.xml (hunk 2 above)
Comment (mkarg) [reply examples/pom.xml:251]: I am +-0 here, as this is just a dependency of some example, and nobody restricts examples to Java EE. What do others think?

**Comment by mkarg** [comment]
> ...it'd be nice of the Travis CI tool ran so that we could ensure that the updated Maven tools don't break us (though I think it is unlikely to do so). Thanks!

Please see https://bugs.eclipse.org/bugs/show_bug.cgi?id=574335.

File: examples/pom.xml (hunk 2 above)
Comment (mkarg) [reply examples/pom.xml]: @andymc12  Replace 4.0.0.Alpha1 by 3.0.0 in https://github.com/eclipse-ee4j/jaxrs-api/pull/980/commits/7ff2b9ec95fe7bbc01e9a42fad2028a548b29d71. Fine?

**Review by andymc12 — APPROVED** [review]
yep - thanks for switching that.  LGTM.

**Review by chkal — APPROVED** [review]

**Comment by mkarg** [comment]
Travis cancelled the build due to missing credits, but the EF admins say we do have 25000 credits. This is really PITA. Maybe we should switch over to EF-hosted Jenkins for PR builds.

**Comment by andymc12** [comment]
> Travis cancelled the build due to missing credits, but the EF admins say we do have 25000 credits. This is really PITA. Maybe we should switch over to EF-hosted Jenkins for PR builds.

Pardon the naiveté, but would we run into similar issues with Jenkins?  Assuming no,  I'm fine with switching over to Jenkins - I've used it in other OSS projects and it seems comparable to Travis.

**Comment by mkarg** [comment]
> > Travis cancelled the build due to missing credits, but the EF admins say we do have 25000 credits. This is really PITA. Maybe we should switch over to EF-hosted Jenkins for PR builds.
> 
> Pardon the naiveté, but would we run into similar issues with Jenkins? Assuming no, I'm fine with switching over to Jenkins - I've used it in other OSS projects and it seems comparable to Travis.

EF-hosted Jenkins is free and unlimited, so no, we won't run into such trouble as it is not Travis-related, it is just related to the fact that the EF admins want us to use EF-hosted Jenkins instead of Travis GmbH hosted Travis.

**Pull request merged**